frontend/components/product/ProductEc.js

```diff
@@ -0,0 +1,82 @@
+import Link from "next/link";
+
+export default function ProductEc({ imgsrc, link, context }) {
+  return (
+    <>
+      <div className="btn">
+        <Link href="/">
+          <a>
+            <div className="pab">
+              <div className="front">
+                <img src="/sample.png" />
+              </div>
+              <div className="back">
+                <p class="bottom_title">test</p>
+                <p class="bottom_detail">test_detail</p>
```

Comment:
```suggestion
                <p className="bottom_title">test</p>
                <p className="bottom_detail">test_detail</p>
```

여기 `className`으로 바꿔야할 것 같아요!